cpp/arcticdb/CMakeLists.txt

```diff
@@ -90,6 +90,10 @@ else()
     list(REMOVE_ITEM NEW_FOLLY_DEPS "/usr/local/opt/openssl/include" )
     set_target_properties(Folly::folly_deps PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "${NEW_FOLLY_DEPS}")
 
+    add_library(xxHash STATIC IMPORTED)
+    set_target_properties(xxHash PROPERTIES IMPORTED_LOCATION ${xxHash_LIBRARY})
+    set_target_properties(xxHash PROPERTIES INTERFACE_INCLUDE_DIRECTORIES ${xxHash_INCLUDE_DIR})
```

Comment:
Shouldn't this be done for conda packages and for wheels (i.e. independently from `ARCTICDB_USING_CONDA`)?

Reply:
`xxHash` isn't a target in the vcpkg build, so this won't work